pkg/prober/prober.go

```diff
@@ -37,6 +37,19 @@ type Prober struct {
 func NewProber(component component.Component, logger log.Logger, reg prometheus.Registerer) *Prober {
 	initialErr := fmt.Errorf(initialErrorFmt, component)
 
+	// From Kubernetes documentation https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/ :
```

Comment:
Minor nit but this should part of struct TBH (for GoDocs)

Reply:
good point, is it ok this way?